### PR TITLE
Conda Bump for base image

### PR DESCRIPTION
**Pull request by apeltzer**
Bump to latest stable conda 4.8.3 for next tools release to get rid of warning.

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

### CODE REVIEW

**Comment by ewels** [comment]
I take it that there isn’t an updated version of the base image? That would be easier than updating conda ourselves..

**Comment by apeltzer** [comment]
Unfortunately its not been updated since 8months on their side :-(

**Comment by ewels** [comment]
Really? What's `continuumio/miniconda3:4.8.2` then? https://hub.docker.com/layers/continuumio/miniconda3/4.8.2/images/sha256-456e3196bf3ffb13fee7c9216db4b18b5e6f4d37090b31df3e0309926e98cfe2?context=explore

**Comment by apeltzer** [comment]
Yeah, I've apparently been looking at this one: https://hub.docker.com/r/continuumio/miniconda 

and ignoring the "miniconda**3**" 🤦

**Comment by apeltzer** [comment]
Adjusted accordingly, thanks for the hint @ewels ;-)

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +34,7 @@
 * Move some of the issue and PR templates into HTML `<!-- comments -->` so that they don't show in issues / PRs
 * Added `macs_gsize` for danRer10, based on [this post](https://biostar.galaxyproject.org/p/18272/)
 * nf-core/tools version number now printed underneath header artwork
+* Bumped Conda version shipped with nfcore/base to 4.8.3
```
Comment (ewels) [inline CHANGELOG.md:37]:
```suggestion
* Bumped Conda version shipped with nfcore/base to 4.8.2
```

**Pull request merged**